Source/WebKit/Platform/spi/ios/UIKitSPI.h

```diff
@@ -1280,6 +1280,7 @@ extern const NSString *UIPreviewDataAttachmentIndex;
 extern NSString * const UIPreviewDataAttachmentListIsContentManaged;
 
 UIEdgeInsets UIEdgeInsetsAdd(UIEdgeInsets lhs, UIEdgeInsets rhs, UIRectEdge);
+UIEdgeInsets UIEdgeInsetsSubtract(UIEdgeInsets lhs, UIEdgeInsets rhs, UIRectEdge edges);
```

Comment:
Is this declaration used anywhere?

Reply:
Nevermind, this was used in a previous iteration of the patch. No longer needed.